src/Menu.php

```diff
@@ -123,14 +123,16 @@ public function addGroup($name, string $template = 'menugroup.html')
             $name = [$name];
         }
 
-        $title = $name['title'] ?? $name['text'] ?? $name['name'] ?? $name[0] ?? null;
+        $title = $name['title'] ?? $name[0] ?? null;
 
         if ($title !== null) {
             $group->template->set('title', $title);
         }
 
         if (isset($name['icon'])) {
             Icon::addTo($group, [$name['icon']], ['Icon'])->removeClass('item');
+        } elseif (isset($name['label'])) {
```

Comment:
> Caveat: It would be better if the items were right-aligned (the icon or the label) - for menu items (see below), they are right aligned, so the label spreads out to the left. If anyone has a hint, how to set classes for that please contribute.

The solution is probably to implement the "menu group item" the same way as "menu item", currently the HTML/CSS for "menu group item" is using CSS/FUI columns, but "menu item" right float positioning.

Reply:
in https://github.com/atk4/ui/pull/1847/commits/93ac62f1aa8a1bd2678138e2836d9fb99bbe55c5 I have commit debug code to debug/unify this